docs/contributing/docs-contributions/docs-components.md

```diff
@@ -92,7 +90,7 @@ Rendered, it looks like this:
 
 ### Pull Quote
 
-The `<Pullquote />` component is used to call out a quote in the blog. It applies borders and styles that make a section of the content pop out to readers.
+The `<Pullquote />` component is used to call out a quote. It applies borders and styles that make a section of the content pop out to readers.
 
 #### Usage
 
```

Comment:
References to the blog in the hidden lines below:

> This prop ~~not used in the blog to help the quote stand out, but~~ could be used in docs where it's necessary to keep content from overlapping with other sections of the layout, such as the Table of Contents.

Also code block example is for blog, not docs